airflow/www/static/js/tree/ResetRoot.jsx

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { Button, Link } from '@chakra-ui/react';
+
+import { getMetaValue } from '../utils';
+
+const dagId = getMetaValue('dag_id');
+const numRuns = getMetaValue('num_runs');
+const urlRoot = getMetaValue('root');
+const baseDate = getMetaValue('base_date');
+
+const ResetRoot = () => {
+  let rootLink;
+  if (urlRoot) {
+    const params = new URLSearchParams();
+    if (numRuns) params.set('num_runs', numRuns);
+    if (baseDate) params.set('base_date', baseDate);
+    rootLink = `/dags/${dagId}/grid?${params.toString()}`;
```

Comment:
We can't hard code the URL like this - airflow can be rooted/deployed to a location other than `/`.

In other places we put the URL in a meta tag from the jinja templates for this purpose.

Reply:
All urls in react should be updated now.